### PR TITLE
implemented functionality to show alert when user changes facility or store ( #1yzhpm7 ).

### LINKED ISSUE #114: On store change current search and upload list should be cleared
When the user changes the store, there should be an alert that it will clear the current upload list and current product search & upload list should be cleared

### PR DESCRIPTION
**Pull request by meet-aniket**
closes #114

### CODE REVIEW

File: src/store/modules/product/mutations.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ const mutations: MutationTree <ProductState> = {
   [types.PRODUCT_CLEAR_UPLD_PRDTS] (state) {
     state.uploadProducts = {};
   },
+  [types.PRODUCT_CLEAR_SEARCH_PRDTS] (state) {
```
Comment (ymaheshwari1) [inline src/store/modules/product/mutations.ts:23]: We don't need to create another mutation to clear search list, we can use the `PRODUCT_SEARCH_UPDATED` mutation to clear the search product by passing empty values to the products and totalProductsCount.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue [hunk 2]
```diff
@@ -63,44 +63,69 @@ export default defineComponent({
...
+            return true;
+          }
         }, {
           text: this.$t('Ok'),
           handler: () => {
-            this.store.dispatch('product/clearUploadProducts');
-            this.store.dispatch('user/logout').then(() => {
-              this.router.push('/login');
-            })
+              this.store.dispatch('product/clearUploadProducts');
```
Comment (bashu22tiwari) [inline src/views/Settings.vue:94]: Improve indentation.

Comment (meet-aniket) [reply src/views/Settings.vue:96]: Improved @ymaheshwari1 Sir, please take a look.

File: src/store/modules/product/mutations.ts (hunk 1 above)
Comment (meet-aniket) [reply src/store/modules/product/mutations.ts]: Improved code please take a look Sir.

**Comment by disha1202** [comment]
Not needed, closing.

**Pull request closed without merging**